db/migrate/20220301110553_add_metadata_to_user_cues.rb

```diff
@@ -0,0 +1,5 @@
+class AddMetadataToUserCues < ActiveRecord::Migration[6.1]
+  def change
+    add_column :user_cues, :metadata, :text
```

Comment:
Cool! Just checking do we need to do anything else than specifying :metadata it as :text?
For the JSON data pair to work? I guess no but just double-checking